datasets/stereoset/stereoset.py

```diff
@@ -0,0 +1,132 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Lint as: python3
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_CITATION = """\
+@article{nadeem2020stereoset,
+  title={StereoSet: Measuring stereotypical bias in pretrained language models},
+  author={Nadeem, Moin and Bethke, Anna and Reddy, Siva},
+  journal={arXiv preprint arXiv:2004.09456},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+StereoSet is a dataset that measures stereotype bias in language models. StereoSet consists of 17,000 sentences that
+measures model preferences across gender, race, religion, and profession.
+"""
+
+_LICENSE = "CC BY-SA 4.0"
+
+
+class StereoSetConfig(datasets.BuilderConfig):
+    """BuilderConfig"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(StereoSetConfig, self).__init__(**kwargs)
+
+
+class StereoSet(datasets.GeneratorBasedBuilder):
+
+    _DOWNLOAD_URL = "https://github.com/moinnadeem/StereoSet/raw/master/data/dev.json"
+
+    BUILDER_CONFIGS = [
+        StereoSetConfig(
+            name="intersentence", version=datasets.Version("1.0.0"), description="intersentence task for StereoSet"
+        ),
+        StereoSetConfig(
+            name="intrasentence", version=datasets.Version("1.0.0"), description="intrasentence task for StereoSet"
+        ),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # datasets.features.FeatureConnectors
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "target": datasets.Value("string"),
+                    "bias_type": datasets.Value("string"),
+                    "context": datasets.Value("string"),
+                    "sentences": datasets.features.Sequence(
+                        {
+                            "sentence": datasets.Value("string"),
+                            "id": datasets.Value("string"),
+                            "labels": datasets.features.Sequence(
+                                {
+                                    "label": datasets.Value("string"),
+                                    "human_id": datasets.Value("string"),
+                                }
+                            ),
+                            "gold_label": datasets.Value("string"),
```

Comment:
Can you use the ClassLabel feature type here ?

For example
```
"gold_label": datasets.ClassLabel(names=["anti-stereotype", "stereotype", "unrelated", "related"])
```

you'll need to regenerate the dataset_infos.json file after this change

Reply:
I replaced `datasets.Value("string")` with `datasets.ClassLabel(names=["anti-stereotype", "stereotype", "unrelated"])` for both `label` of individual humans and `gold_label`. (there is no "related" value for both columns)